src/core/launcher/index.ts

```diff
@@ -22,22 +24,31 @@ export class Launcher extends Observable implements DefaultLauncher {
   private activeComponents: ComponentNames[] = [];
   private componentsToAttachAfterJoin: Partial<BaseComponent>[] = [];
   private activeComponentsInstances: Partial<BaseComponent>[] = [];
-  private participant: Participant;
-  private group: Group;
 
   private realtime: AblyRealtimeService;
   private eventBus: EventBus = new EventBus();
 
-  private participants: Participant[] = [];
-  constructor({ participant, group }: LauncherOptions) {
+  private participant: PublicSubject<Participant>;
+  private participants: PublicSubject<Participant[]>;
+  private group: PublicSubject<Group>;
+
+  constructor({ participant, group: participantGroup }: LauncherOptions) {
     super();
+    const { localParticipant, participants, group } = useGlobalStore();
+
+    this.participant = localParticipant;
+    this.participants = participants;
+    this.group = group;
 
-    this.participant = {
+    setTimeout(() => {
+      localParticipant.value = { ...localParticipant.value, color: 'red' };
+    }, 20000);
```

Comment:
It's a mock, right?! You shouldn't need to update the participant after 20s and assign red color to it.

Reply:
Yes, it is a mock